cmd/wego/flux/cmd.go

```diff
@@ -0,0 +1,34 @@
+package flux
+
+import (
+	_ "embed"
+	"fmt"
+	"os"
+	"os/exec"
+
+	"github.com/spf13/cobra"
+)
+
+//go:embed flux
+var flux []byte
```

Comment:
omg, so simple. That's awesome.

One thing tho, since the dependencies script downloads them to `tools/bin`, do you mind changing to pointing to there?

take a look at https://github.com/weaveworks/weave-gitops/pull/43 it should be merged shortly.

Reply:
Okay so thats going to be an interesting thing to do. With go embed i am not allowed to use a file whose path has ../ I am unable to go outside the package. So I can move all of this to the root but that may not be what we want.